PovioKit/Classes/Extensions/UIKit/UIView+Povio.swift

```diff
@@ -8,6 +8,12 @@
 
 import UIKit
 
+public extension UIView {
```

Comment:
@borut-t Re-added this, otherwise, the code doesn't compile.

Reply:
Why do we need it? I think we are already covered for the table/collection views reusable identifiers 🤷‍♂

Reply:
`UICollectionView` and `UITableView` extension methods have methods like
```
  func register<T: UICollectionViewCell>(_: T.Type) {
    register(T.self, forCellWithReuseIdentifier: T.identifier)
  }
```
which use `.identifier` static prop.

Reply:
Ok, so where do we need it?

Reply:
Have you tried compiling the code?

Reply:
I haven't really noticed we have an example app 🙈